src/main/java/controller/InputController.java

```diff
@@ -0,0 +1,36 @@
+package controller;
+
+import static utils.TryCountValidator.validateTryCountFormat;
+import static view.InputView.inputCarName;
+import static view.InputView.inputTryCount;
+
+import model.Race;
+
+public class InputController {
+
+    public Race makeRace() {
+        try {
+            String carNames = inputCarName();
+            Race race = new Race(carNames);
+
+            return race;
```

Comment:
이런 경우에는 바로 return을 해주시면 이름 고를 일이 하나 줄어듭니다😁
```suggestion
            return new Race(carNames);
```